packages/dds/tree/src/test/simple-tree/objectFactory.spec.ts

```diff
@@ -19,35 +20,37 @@ import { getRoot } from "./utils.js";
 describe("SharedTreeObject factories", () => {
 	const sb = new SchemaFactory("test");
 
-	class ChildA extends sb.object("childA", {
+	const ChildA = sb.object("childA", {
 		content: sb.number,
-	}) {}
+	});
```

Comment:
These were gratuitously converted to classes.  Converting them back so I can re-enable the tests below.